package.json

```diff
@@ -22,7 +22,11 @@
   "author": "Angular2-Meteor (https://github.com/Urigo/angular2-meteor)",
   "license": "MIT",
   "scripts": {
-    "build": "gulp build",
+    "typings": "typings i",
+    "prebuild": "npm run typings && npm run lint",
+    "build": "npm run build-only",
+    "build-only": "tsc",
```

Comment:
Does `npm build` fail for you? Cause it does for me.
TSC throws some compilation warnings/errors (some of them is unavoidable now)
So, for better output control it make sense to exec, for example, tsc in the node.